cves/CVE-2019-14232.yml

```diff
@@ -268,7 +291,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Lack of extensive testing of edge cases is what caused this problem to arrise.
+    The vulnerability is exposed when a default parameter was passed a different value.
```

Comment:
Would be good to mention what the regular expression failed to accomplish that led to the vulnerability

Reply:
Or briefly explain the part of the regex that was backtracking